robosuite/demos/demo_segmentation.py

```diff
@@ -0,0 +1,107 @@
+"""
+Play random actions in an environment and render a video that demonstrates segmentation.
+"""
+import imageio
+import colorsys
+import random
+import numpy as np
+import matplotlib.cm as cm
+
+import robosuite as suite
+from robosuite.controllers import load_controller_config
+
+
+def random_colors(N, bright=True):
+    """
+    Modified from https://github.com/matterport/Mask_RCNN/blob/master/mrcnn/visualize.py#L59
+    Generate random colors.
+    To get visually distinct colors, generate them in HSV space then
+    convert to RGB.
+    """
+    brightness = 1.0 if bright else 0.7
+    hsv = [(i / N, 1, brightness) for i in range(N)]
+    colors = np.array(list(map(lambda c: colorsys.hsv_to_rgb(*c), hsv)))
+    rstate = np.random.RandomState(seed=0)
+    rstate.shuffle(colors)
+    return colors
+
+def segmentation_to_rgb(seg_im):
+    """
+    Helper function to visualize segmentations as RGB frames.
+    NOTE: assumes that geom IDs go up to 255 at most - if not,
+    multiple geoms might be assigned to the same color.
+    """
+    # ensure all values lie within [0, 255]
+    seg_im = np.mod(seg_im, 256)
+
+    # colors = random_colors(N=256, bright=True)
```

Comment:
we can probably remove this and the random_colors function right? It was there for debugging purposes